tests/functional/glusterd/test_brickmux_brick_process.py

```diff
@@ -0,0 +1,92 @@
+#  Copyright (C) 2018-2021  Red Hat, Inc. <http://www.redhat.com>
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+from glusto.core import Glusto as g
+from glustolibs.gluster.exceptions import ExecutionError
+from glustolibs.gluster.gluster_base_class import GlusterBaseClass
+from glustolibs.gluster.volume_libs import bulk_volume_creation, cleanup_volume
+from glustolibs.gluster.volume_ops import (
+    set_volume_options,
+    get_volume_list,
```

Comment:
This can be written in a single line, why this kind of formatting?

Reply:
Yes I agree this can be  written in a single line, for readability I wrote like this, plz let me know  if I need to change

Reply:
I would suggest to keep it in a single like doesn't really make a difference.

Reply:
sure will make the changes Thanks